packages/lab/src/dialog-next/DialogNext.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  forwardRef,
+  HTMLAttributes,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import { clsx } from "clsx";
+import {
+  FloatingFocusManager,
+  FloatingOverlay,
+  FloatingPortal,
+} from "@floating-ui/react";
+import { makePrefixer, useForkRef, useId } from "@salt-ds/core";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useDialogNext } from "./useDialogNext";
+
+import dialogNextCss from "./DialogNext.css";
+import { DialogNextContext } from "./DialogNextContext";
+import { DialogNextBody } from "./DialogNextBody";
+
+export interface DialogNextProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Display or hide the component.
+   */
+  open?: boolean;
+  /**
+   * Callback function triggered when open state changes.
+   */
+  onOpenChange?: (open: boolean) => void;
+  /**
+   * Status indicator
+   * */
+  status?: "info" | "success" | "warning" | "error";
+}
+
+const withBaseName = makePrefixer("saltDialogNext");
+
+export const DialogNext = forwardRef<HTMLDivElement, DialogNextProps>(
+  function DialogNext(props, ref) {
+    const {
+      children,
+      className,
+      open = true,
```

Comment:
The default is to have the dialog be open?

Reply:
I based this component off of Drawer. I didn't notice this but I assume it was intentional there. I'll double check with Jo.

Reply:
They thought it was a good default for Drawer so I will leave it the same in Dialog. But I don't think I have ever used a Dialog without controlling the open state so not sure it matters much.